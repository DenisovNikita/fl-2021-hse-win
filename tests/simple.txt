0 to 1 by 5 0____1
1
2

