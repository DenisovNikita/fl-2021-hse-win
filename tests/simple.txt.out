LexToken(NUM,0,1,0)
LexToken(TO,'to',1,2)
LexToken(NUM,1,1,5)
LexToken(BY,'by',1,7)
LexToken(NUM,5,1,10)
LexToken(CHAR,'0____1',1,12)
LexToken(NUM,1,2,19)
LexToken(NUM,2,2,21)
